src/main/java/training/commands/elbow/MoveElbowToAngle.java

```diff
@@ -0,0 +1,32 @@
+package training.commands.elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.elbow.Elbow;
+
+public class MoveElbowToAngle extends Command {
+
+	private Rotation2d targetAngle;
+
+	public Elbow getInstance() {
+		return Elbow.getInstance();
+	}
+
+	public MoveElbowToAngle(Rotation2d targetAngle) {
+		this.targetAngle = targetAngle;
+	}
+
+	@Override
+	public void initialize() {
```

Comment:
consider moving this to the execute, since it needs to update the pid every 20 milisecnods